airbyte-cdk/python/unit_tests/sources/streams/http/error_handlers/test_http_status_error_handler.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from unittest.mock import MagicMock
+
+import pytest
+import requests
+from airbyte_cdk.models import FailureType
+from airbyte_cdk.sources.streams.http.error_handlers import HttpStatusErrorHandler, ResponseAction
+
+logger = MagicMock()
+
+def test_given_ok_response_http_status_error_handler_returns_none():
+    response = requests.Response()
+    response.status_code = 200
```

Comment:
should there also be a test for `201`?

Reply:
The test is really that the interpret_response doesn't return a retry/fail/ignore action. I updated to mock the response.ok method to return true. (response.ok by default returns true 200 <= status < 400)